app/swagger/swagger/schemas/appeals/contestable_issues.rb

```diff
@@ -11,7 +11,7 @@ class ContestableIssues
             items type: :object do
               key :description, 'A contestable issue (to contest this, you include it as a RequestIssue'\
                                 ' when creating a HigherLevelReview, SupplementalClaim, or Appeal)'
-              property :type, type: :string, enum: %w[ContestableIssue]
```

Comment:
we probably want to make corresponding changes to https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/20-0996/schema.js